src/acbm/preprocessing.py

```diff
@@ -416,7 +414,7 @@ def add_location(
 
 
 def add_locations_to_activity_chains(
-    activity_chains: pd.DataFrame, target_crs: str
+    activity_chains: pd.DataFrame, target_crs: str, centroid_layer: pd.DataFrame
```

Comment:
Perhaps make this not an arg but a class method on config so it can be read within the function as before

Reply:
Opening a new issue to update this API and remove `root_path` from config.